sorting-algos/merge/mergesort.py

```diff
@@ -1 +1,49 @@
-"""The merge sorting algorithm."""
\ No newline at end of file
+"""The merge sorting algorithm."""
+
+
+def mergesort(unsorted):
+    """Take in an unsorted list and sort it recursivly using the merge sort method."""
+    if len(unsorted) < 2:
+        return unsorted
+
+    left = []
+    right = []
+    count = 0
+
+    for item in unsorted:
```

Comment:
If you want both the item and the index for a list, it is more pythonic to use `enumerate()`.